RecoJets/JetProducers/plugins/FastjetJetProducer.cc

```diff
@@ -520,6 +455,97 @@ void FastjetJetProducer::runAlgorithm( edm::Event & iEvent, edm::EventSetup cons
+	descFastjetJetProducer.add<int>("maxDepth",	-1);
+	descFastjetJetProducer.add<int>("nFilt",	-1);
+	descFastjetJetProducer.add<int>("MinVtxNdof",	5);
+	///// From VirtualJetProducer
+	descFastjetJetProducer.add<string> ("@module_label",	"" );
+	descFastjetJetProducer.add<InputTag>("src",	InputTag("particleFlow") );
+	descFastjetJetProducer.add<bool>("doAreaFastjet",	false );
+	descFastjetJetProducer.add<double>("Rho_EtaMax", 	4.4 	);
+	descFastjetJetProducer.add<double>("rParam",		0.4 );
+	descFastjetJetProducer.add<string>("jetAlgorithm",	"AntiKt" );
+	descFastjetJetProducer.add<InputTag>("srcPVs",	InputTag("") );
+	descFastjetJetProducer.add<string>("jetType",		"PFJet" );
+	descFastjetJetProducer.add<double>("inputEtMin", 	0.0 );
+	descFastjetJetProducer.add<double>("inputEMin",		0.0 );
+	descFastjetJetProducer.add<double>("jetPtMin",		5. );
+	descFastjetJetProducer.add<bool>("doPVCorrection",	false );
+	descFastjetJetProducer.add<bool>("doRhoFastjet",	false );
+	descFastjetJetProducer.add<string>("jetCollInstanceName", ""	);
+	descFastjetJetProducer.add<bool>("doPUOffsetCorr", 	false	);
+	descFastjetJetProducer.add<string>("subtractorName", 	""	);
+	descFastjetJetProducer.add<bool>("useExplicitGhosts", 	false	);
+	descFastjetJetProducer.add<bool>("doAreaDiskApprox", 	false 	);
+	descFastjetJetProducer.add<double>("voronoiRfact", 	-0.9 	);
+	descFastjetJetProducer.add<double>("Ghost_EtaMax",	5. 	);
+	descFastjetJetProducer.add<int>("Active_Area_Repeats",	1 	);
+	descFastjetJetProducer.add<double>("GhostArea",	 	0.01 	);
+	descFastjetJetProducer.add<bool>("restrictInputs", 	false 	);
+	descFastjetJetProducer.add<unsigned int>("maxInputs", 	1 	);
+	descFastjetJetProducer.add<bool>("writeCompound", 	false 	);
+	descFastjetJetProducer.add<bool>("doFastJetNonUniform", false 	);
+	descFastjetJetProducer.add<bool>("useDeterministicSeed",false 	);
+	descFastjetJetProducer.add<unsigned int>("minSeed", 	14327 	);
+	descFastjetJetProducer.add<int>("verbosity", 		0 	);
+	descFastjetJetProducer.add<double>("puWidth",	 	0. 	);
+	descFastjetJetProducer.add<unsigned int>("nExclude", 	0 	);
+	descFastjetJetProducer.add<unsigned int>("maxBadEcalCells", 	9999999	);
+	descFastjetJetProducer.add<unsigned int>("maxBadHcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxProblematicEcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxProblematicHcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxRecoveredEcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxRecoveredHcalCells",	9999999 );
+	vector<double>  puCentersDefault;
+	descFastjetJetProducer.add<vector<double>>("puCenters", 	puCentersDefault);
+	///// From PileUpSubstractor
+	descFastjetJetProducer.add<double> ("puPtMin", 	10.);
+	descFastjetJetProducer.add<double> ("nSigmaPU", 	1.);
+	descFastjetJetProducer.add<double> ("radiusPU", 	0.5);
+	descFastjetJetProducer.add<bool> ("sumRecHits", 	false);
```

Comment:
@perrotta unfortunately the "sumRecHits" is needed in the FastjetJetproducer. The only way to get away from the problems here  https://github.com/cms-sw/cmssw/pull/18832#issuecomment-303207499 is to hardcode in the fillDescriptions. Unless you have something else to recommend.